### DISCUSSION TITLE
resolve workitem link direction?

### DISCUSSION THREAD

**Discussion by bigblondewolfpersonal** [jesper-raemaekers/python-polarion#130] [Q&A]
Hi, how do I see the direction of a workitem link? When I use getLinkedItemWithRoles(), the link role will be "parent" whether it is "has parent" or "is parent of".
I'd like to get a hierarchical tree of workitems e.g. requirements for a block given the block's master workitem - how can I trace this relationship down?
Thanks,
Oggy

**Comment by dkaltmann** [comment]
Came across this old thread while googling, I'd also like to know if there is a way to figure out if there's a way to tell the direction of workitem link please 🙏

**Comment by Polarator** [comment]
I have a similar goal as Oggy. 
Has anyone a idea to get the direction of a link?

Edit:
It is not a problem with the Python library. The TrackerWebService has no service for getting only direct or back links. In Polarion the TrackerService supports getLinkedWorkItemsBack() and getLinkedWorkItemsDirect() additionally to getLinkedWorkItems(), which returns all linked Work Items.
